tests/test_users.py

```diff
@@ -33,6 +33,7 @@ def GET_USER_PUBLIC():
 
 @fixture()
 def clean_users_table():
+    delete_all_users()
```

Comment:
Må det slettes to ganger?

Reply:
pga. `yield` på linje 37 så vil den første sletting skje før test funksjonen kjører, også etter `yield` skjer etter test funksjonen har kjørt.
Det er bare hel-gardering :^)